src/App.tsx

```diff
@@ -40,7 +40,7 @@ function App() {
       setRows(
         getPeopleQuery.data
           .map((person, index) => ({ id: index, ...person })) // Add a unique ID to each row
-          .sort((a, b) => a.name.localeCompare(b.name)), // Sort by name
+          .sort((a, b) => a.name.localeCompare(a.name)), // Sort by name
```

Comment:
**Suggestion:** Correct the sorting logic to compare 'a' and 'b' instead of 'a' and 'a'.
```suggestion
          .sort((a, b) => a.name.localeCompare(b.name)), // Sort by name
```